src/pages/solve-view/SolveView.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import CodeEditor from '../../components/code-editor';
+import Console from '../../components/console';
+import Nav from '../../components/nav';
+import { useParams } from 'react-router-dom';
+import { PageContainer } from '../../components/base';
+import MarkdownEditor from '../../components/markdown-editor';
+const SolveView = () => {
+  const { id } = useParams();
+  return (
+    <PageContainer >
+      <Nav />
+      <div className={styles['col-container']}>
+        <div className={styles.column}>
+          <MarkdownEditor edition="disabled"/>
```

Comment:
why use edition as a string if you inside the component are using a bool variable? 

we can use `disabled={true}` and use this property inside the component